go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	github.com/secure-systems-lab/go-securesystemslib v0.4.0
 	github.com/sigstore/fulcio v1.0.0
 	github.com/sigstore/rekor v1.0.1
-	github.com/sigstore/sigstore v1.5.0
+	github.com/sigstore/sigstore v1.5.1-0.20230117171511-8134411c70e1
```

Comment:
let me cut a sigstore release

Reply:
thanks!

Reply:
you can use https://github.com/sigstore/sigstore/releases/tag/v1.5.1

Reply:
you're awesome, thanks :)